doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -110,41 +113,53 @@ The presence of a ``bicr`` node in the application devicetree will automatically
 Flashing the BICR binary
 ========================
 
+##TODO
+
 After the |NCS| build system generates the BICR binary, you must flash this binary manually.
 The content of BICR should be loaded to the SoC only once and should not be erased nor modified unless the PCB layout changes.
-To manually flash the generated :file:`bicr.hex` file to the SoC, use nRF Util as follows::
+To manually program the generated :file:`bicr.hex` file to the SoC, use nRF Util as follows::
+
+    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex --core Application --serial-number <serial_number>
+
+You only need to follow this programming process once, assuming the PCB configuration applied through the BICR is correct the first time.
+However, it is also possible to reprogram the BICR while in the LCS ``RoT``.
+This can be useful, for example, when adjusting the configuration as the PCB design gets refined or modified, requiring the process to be repeated.
+
+Validate the BICR binary
+------------------------
+
+After programming the BICR binary onto the device, you should validate whether the BICR was successfully installed and executed.
 
-    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex` --core Secure --serial-number <serial_number>
+Reset the device
+   The simplest way to check if the BICR is functioning is to reset the device.
+   If the device is still operational after the reset, the BICR is at least partially loaded.
```

Comment:
When I read this as a user, the first thing I think is: "... and what if it is not operational?"

Reply:
@hellesvik-nordic Sure, that makes sense. Could you describe how to detect that?

Reply:
I do not really know how to detect that. I have not written wrong BICR to a board yet.
Is there maybe a specific nrfutil error message that would pop up if BICR is broken? 
Maybe something along the lines of "If after a reset, nrfutil gives the error [INSERT ERROR], meaning that Jlink cannot find the device, then the device is broken because the BICR was wrong.".
I too hope Inge can help getting this right.

Reply:
Yeah, the problem is that if the BICR is invalid then the chip can be in an undefined state, so almost anything can happen, and almost any error can be returned. It would of-course depend on the specifics of the board, and the BICR. I dont remember the exact messages offhand, but this is a list roughly in order, which would all have separate errors:

* JLink error due to bad voltage on the SWD port
* DAP fails to power up - probably the most likely one, I have seen these on kits more than once
* Failed to identify the device - indicating other power issues in the DAP
* Access ports fail to open - This indicates that the SDROM crashed before it was able to open the Secure Domain access port
* Memory access errors when accessing peripherals
* MRAM read/write errors

Reply:
1) Is the inductor really needed? YES. It is needed and not optional.
2) Why does BICR.POWER.CONFIG:INDUCTOR not make any sense to you? You must place the inductor in the board and you must write in the BICR that the inductor is present.

Reply:
Since proving the negative here will be hard, I think trying to describe it can cause confusion.
Therefore I will close this conversation, but we can keep it in mind for later to maybe improve this when we got more experience with users trying to follow the guide